frontend/app/services/EventbriteService.scala

```diff
@@ -122,25 +138,22 @@ abstract class EventbriteService(implicit val ec: ExecutionContext, system: Acto
   def getOrder(id: String): Future[EBOrder] = get[EBOrder](s"orders/$id/", "expand" -> EBOrder.expansions.mkString(","))
 }
 
-abstract class LiveService(implicit ec: ExecutionContext, system: ActorSystem, contentApiService: GuardianContentService, gridService: GridService) extends EventbriteService {
```

Comment:
this liveservice class looked like it was there for existing/old reasons so I got rid of it.  Although given @rupertbates comments last week about assuming things are needed, I'm not sure it's a good idea!  But this time I'm more confident it's not needed, last time I was quite uncertain.